File: 268-MissingNumber/268-MissingNumber.py
# Last updated: 2/3/2026, 9:39:37 PM
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
      n=len(nums)
      x=n
      for i in range(n):
        x=x^i^nums[i]
      return x
        

File: 268-MissingNumber/268-MissingNumber.java
// Last updated: 2/3/2026, 9:39:45 PM
class Solution {
    public int missingNumber(int[] nums) {
        int sum=Arrays.stream(nums).sum();
        int n=nums.length;
        int cal=n*(n+1)/2;
        int result=cal-sum;
      return result;
    }
}